.github/workflows/test.yml

```diff
@@ -59,8 +59,9 @@ jobs:
       -
         name: Codecov
         if: matrix.go == '1.18'
-        uses: codecov/codecov-action@v1.2.1
+        uses: codecov/codecov-action@v2
         with:
-          file: ./coverage.out # optional
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
after reading the documentation, it's still not clear to me why a token would be required for public repo.

Reply:
It didn't work without 🤷‍♂️